README.md

```diff
@@ -18,6 +18,23 @@ then create a new document
     doc = Govspeak::Document.new "^Test^"
     puts doc.to_html
 
+## Changes appearing across GOV.UK
+
+Some additional steps or considerations are needed to ensure changes to govspeak cascade across GOV.UK in a holistic way.
+
+Once govspeak has been updated and version incremented then: 
+- [`govuk_publishing_components` govspeak](https://components.publishing.service.gov.uk/component-guide/govspeak) will also need updating to reflect your most recent change.
+- [Publishing apps](https://docs.publishing.service.gov.uk/apps.html) (including but not limited to [content-publisher](https://github.com/alphagov/content-publisher) & [whitehall](https://github.com/alphagov/whitehall)) also use govspeak, these apps will need to be released with the new govspeak version present.
```

Comment:
Not a blocker at all. You can use [reference links] in GitHub flavoured markdown by going `[whitehall]`, and then in the end of the markdown document go `[whitehall]: https://github.com/alphagov/whitehall`. I mention this because then you can define the link once, and use it everywhere it's needed.

[reference links]: https://www.markdownguide.org/basic-syntax#reference-style-links

Reply:
Ah good to know!